jenkins/internal-ci/generic/pr/cortx_deployment/ha.groovy

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env groovy
+// CLEANUP REQUIRED
+pipeline { 
+    agent {
+        node {
+            label "docker-${OS_VERSION}-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        disableConcurrentBuilds()
+        ansiColor('xterm')  
+    }
+
+    parameters {  
+        string(name: 'HA_URL', defaultValue: 'https://github.com/Seagate/cortx-ha', description: 'Repo for HA')
+        string(name: 'HA_BRANCH', defaultValue: 'main', description: 'Branch for HA')
+        string(name: 'CORTX_RE_URL', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repo for cortx-re')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch for cortx-re')
+
+        choice (
+            choices: ['all', 'cortx-all' , 'cortx-rgw', 'cortx-data'],
```

Comment:
Remove `cortx-all` and add `cortx-control`

Reply:
Done